### ISSUE TITLE
Reading the counter value from the cache

### ISSUE THREAD

**Issue by LidiaSongstad** [stefanprodan/AspNetCoreRateLimit#369]
Hi -  There are examples to read the policy details from the policy store. Can we read the counter value from the counter store . we are using distributed cache to store the values. 

Thanks,

**Comment by ThomasArdal** [comment]
Did you find a way to do this? I'm thinking about pulling some stats from the store to warn users when getting close to the limit, etc.

**Comment by LidiaSongstad** [comment]
yes.. figured out a way to read from the cache by referring the code to understand how the keys are generated to store the counter value .. 
var bytes = Encoding.UTF8.GetBytes($"crlc_{clientId}_{period}");
			using var algorithm = SHA1.Create();
			var hash = algorithm.ComputeHash(bytes);
			var key = Convert.ToBase64String(hash);
			var rateLimitCounter = (await _cache.GetAsync(key));
			if (rateLimitCounter.HasValue)
			{
				RateLimitCounter value = (await _cache.GetAsync(key)).Value;
}

Hope this helps .. If you found a better way , please shaew..

Thanks,

**Comment by LidiaSongstad** [comment]
You are welcome. Thanks for your suggestion..
